models/models.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+import (
+	"database/sql"
+	"errors"
+)
+
+var (
```

Comment:
can we are remove to this variable re-declare  on every page (module)
i means i want re-usability on every where